src/pycram/object_descriptors/mjcf.py

```diff
@@ -0,0 +1,474 @@
+import pathlib
+
+import numpy as np
+import rospy
+from dm_control import mjcf
+from geometry_msgs.msg import Point
+from typing_extensions import Union, List, Optional, Dict, Tuple
+
+from ..datastructures.dataclasses import Color, VisualShape, BoxVisualShape, CylinderVisualShape, \
+    SphereVisualShape, MeshVisualShape
+from ..datastructures.enums import JointType, MJCFGeomType, MJCFJointType
+from ..datastructures.pose import Pose
+from ..description import JointDescription as AbstractJointDescription, \
+    LinkDescription as AbstractLinkDescription, ObjectDescription as AbstractObjectDescription
+from ..failures import MultiplePossibleTipLinks
+
+try:
+    from multiverse_parser import Configuration, Factory, InertiaSource, GeomBuilder
+    from multiverse_parser import (WorldBuilder,
+                                   GeomType, GeomProperty,
+                                   MeshProperty,
+                                   MaterialProperty)
+    from multiverse_parser import MjcfExporter
+    from pxr import Usd, UsdGeom
+except ImportError:
+    rospy.logwarn("Multiverse not found.")
```

Comment:
misleading. Use logerr or dont log at all since an exception is raised anyways

Reply:
done